woocommerce/woocommerce-admin
# SelectControl: fix display of multiple selections without inline tags.

Fixes the display of selections in `SelectControl`s that allow `multiple` and are not displaying `inlineTags` (and the following console error: `Warning: Failed prop type: Invalid prop `query` of type `object` supplied to `Control`, expected `string`.`)

https://github.com/woocommerce/woocommerce-admin/pull/6847#issuecomment-824382874

Regression introduced in: https://github.com/woocommerce/woocommerce-admin/pull/5848

### Detailed test instructions:

- Go to the Categories Report (or any report that has comparison mode)
- Select "comparison" from the Show menu
- Choose at least one item to compare
- Verify the item renders below the search input
